editoast/openapi.yaml

```
@@ -675,6 +675,59 @@ paths:
                       type: object
                       description: object's schematic in geojson format
 
+  /infra/{id}/search:
```

```suggestion
  /search:
```

If we want to include search for other objects such as rolling stocks (which are not linked to infra). We have to change this endpoint and move some stuff.